### PR TITLE
updated src files

### PR DESCRIPTION
**Pull request by alindner14**

### CODE REVIEW

File: Modules/GLO/include/GLO/VertexingQcTask.h [hunk 1]
```diff
@@ -70,7 +73,12 @@ class VertexingQcTask final : public TaskInterface
   std::unordered_map<o2::MCEventLabel, int> mMapEvIDSourceID; // unordered_map counting the number of vertices reconstructed per event and source (--> MCEventLabel)
   TH1F* mNPrimaryMCEvWithVtx = nullptr;                       // event multiplicity for MC events with at least 1 vertex
   TH1F* mNPrimaryMCGen = nullptr;                             // event multiplicity for all MC events
-  TEfficiency* mVtxEffVsMult = nullptr;                       // for vertex efficiency
+<<<<<<< HEAD
```
Comment (Barthelemy) [inline Modules/GLO/include/GLO/VertexingQcTask.h:76]: there are badly merged items, please have a look and fix them

**Review by Barthelemy — CHANGES_REQUESTED** [review]

File: Modules/GLO/include/GLO/VertexingQcTask.h (hunk 1 above)
Comment (alindner14) [reply Modules/GLO/include/GLO/VertexingQcTask.h]: i did. thank you

**Comment by alindner14** [comment]
Hi @Barthelemy , @knopers8 can you please approve this PR and merge if the tests are successful? I would need these changes to be  merged for the upcoming 1M events (https://alice.its.cern.ch/jira/browse/O2-2679).
Thank you!

**Review by Barthelemy — APPROVED** [review]

**Comment by knopers8** [comment]
Please use a more meaningful PR title in the future :)

**Pull request merged**